### ISSUE TITLE
Blackscreen while loading forge with mods

### ISSUE THREAD

**Issue by Lewyfcb9pro** [PojavLauncherTeam/PojavLauncher#2277]
### Describe the bug

When I try to load into Minecraft with any mods (not like optifine or JEI but the bigger ones) it just doesn't load just stays black even after 20 minutes nothing just black with the controls and if it's important I have a Redmi note 10 pro and gave it 2 GB of Ram but it just doesn't work. It started doing this only after the update where the game directory was changed from games/PojavLauncher/.minecraft to Android/data/net.kdt.pojavlaunch/files.

### Steps To Reproduce

```markdown
I don't know if the bug will appear but just start forge 1.12.2 with any bigger mod like GalactiCraft then it maybe will work maybe not.
```

### Expected Behavior

I expect the game to launch.

### Platform

```markdown
- Device model: Redmi note 10 pro
- CPU architecture: cortex a55 (not sure)
- Android version: 11
- PojavLauncher version: latest
```

### Anything else?

It can't even load only the Micdoodle core mod

**Comment by ZIONJADEHD505** [comment]
Mine just crashes

**Comment by Mathias-Boulay** [comment]
Known, pretty sure that's a duplicate of another issue at this point

**Comment by Lewyfcb9pro** [comment]
What do you mean by another issue. And sorry but I was away and didn't have internet connection

**Comment by GoneSubject** [comment]
> What do you mean by another issue. And sorry but I was away and didn't have internet connection

like there's another issue kinda like this

**Comment by Lewyfcb9pro** [comment]
Ohhhh and what causes it? Or how to fix it?

**Comment by GoneSubject** [comment]
> Ohhhh and what causes it? Or how to fix it?

idk

**Comment by Lewyfcb9pro** [comment]
Fuck

**Comment by GoneSubject** [comment]
idk it's not my problem

**Comment by Lewyfcb9pro** [comment]
Do you know how to downgrade the version of pojavlauncher?

**Comment by GoneSubject** [comment]
wdym

**Comment by Lewyfcb9pro** [comment]
Well there is a update function in Google Play Store so I'm asking how to change the version to an older one of it's possible

**Comment by GoneSubject** [comment]
hm idk

**Comment by Mathias-Boulay** [comment]
Cannot be fixed, require specific texture support
